analysis/db/us_map/choroplethMap.py

```diff
@@ -0,0 +1,62 @@
+import plotly.figure_factory as ff
+from pyprojroot import here
+
+import numpy as np
+import pandas as pd
+
+# ChoroplethMap using FIPS from merged data
+
+df = pd.read_csv('https://github.com/CSSEGISandData/COVID-19/raw/master/csse_covid_19_data/csse_covid_19_time_series/'
+                 'time_series_covid19_confirmed_US.csv')
+loc_df = pd.read_excel(here('./data/db/original/maps/State_FIPS.xlsx'))
+merged_df = pd.merge(loc_df, df, right_on='Admin2', left_on='Name')
```

Comment:
why did you not use or also include `FIPS` as matchign variable?

Reply:
cause their formats were different? and the fips in the first dataset didn't work out for building the map and that's why I merged another table. Or should I make it matching variable anyway?

Reply:
oh just curious becuase you asked about the duplicated columns. if the mating columns contained the same format of values  they could be used as matching keys and you won't have duplicated columns. no need to do extra cleaning work if it doesn't provide anything useful